site/content/blog/2023-07-06-zap-2.13.0/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "ZAP 2.13.0"
+summary: "ZAP 2.13.0 has just been [released](/docs/desktop/releases/2.13.0/), and adds support for HTTP/2, improved authentication handling and Mac Silicon."
+images:
+- https://www.zaproxy.org/blog/2023-07-06-zap-2.13.0/images/blog-2-13-0.png
+type: post
+tags:
+- blog
+- release
+date: "2023-07-06"
+authors:
+    - simon
+---
+ZAP 2.13.0 has just been [released](/docs/desktop/releases/2.13.0/), and adds support for HTTP/2, improved authentication handling, and Mac Silicon.
+
+![ZAPbot with 2.13.0 banner](images/blog-2-13-0.png)
+
+Some of the more significant enhancements include:
+
+### HTTP/2 Support
+
+HTTP/2 is now supported, with no configuration changes required.
+
+If you proxy HTTP/2 traffic through ZAP then ZAP will make the same HTTP/2 requests to the target.
+Any tools that work on proxied requests will also automatically use HTTP/2.
+
+### Improved Authentication Handling
+
+ZAP authentication handling has been significantly overhauled, and ZAP can now auto-authenticate to many web apps by just supplying the URL of the login page along with the credentials.
+
+For more details see:
+
+* [Authentication Tester Dialog](/blog/2023-05-23-authentication-tester/)
+* [Authentication Auto-Detection](/blog/2023-05-02-authentication-auto-detection/)
+
+Look for more blog posts, hopefully in the near future.
+
+### Mac Silicon Support
+
+Mac Silicon is now supported via a new [installer](/download/#main) and in the [Docker images](/download/#docker).
+
+### GitHub Container Registry
+
+As explained in [this blog post](/blog/2023-06-13-ghcr-docker-images/) 
+the ZAP Docker images are now also available in the GitHub Container Registry.
```

Comment:
Is there something like docker hub we can link to? Like a GHCR web interface? (I searched quickly but 🤷‍♂️)

Reply:
Theres https://github.com/orgs/zaproxy/packages but its not great